packages/js/data/src/product-form/selectors.ts

```diff
@@ -15,3 +16,9 @@ export const getProductForm = ( state: ProductFormState ) => {
 	const { errors, ...form } = state;
 	return form;
 };
+
+export type ProductFormSelectors = {
+	getFields: WPDataSelector< typeof getFields >;
+	getField: WPDataSelector< typeof getField >;
+	getProductForm: WPDataSelector< typeof getProductForm >;
+} & WPDataSelectors;
```

Comment:
To be used in the `WCSelectorType` case statement so we can correctly get the types by using the `WCDataSelector`, see the `product-form-fills.tsx` for example.